src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'redux/books/book/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/book/REMOVE_BOOK';
+
+const initialState = [];
+
+const newBook = (id, title, author) => ({
+  type: 'ADD_BOOK',
+  title,
+  author,
+});
+
+const removeBook = (id) => ({
+  type: 'REMOVE_BOOK',
+  id,
+});
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
```

Comment:
- [ ] Depending on the approach you above for my code suggestion above... you did not define `payload` in your **`newBook`** action creator so this would likely throw errors. Instead, you can change to this since what you defined is `author` and `title`

  ```
  return [...state, {action.id, action.title, action.author}]
  ```

Reply:
Thank you for pointing this out. I am working on this change right away.